apps/src/templates/codeReviewGroups/codeReviewGroupManagerHooks.js

```diff
@@ -0,0 +1,95 @@
+import {useState} from 'react';
+import _ from 'lodash';
+
+export function useGroups(initialGroups) {
+  const [groups, setGroupsState] = useState(initialGroups);
+
+  const getGroup = droppableId =>
+    _.find(groups, group => group.droppableId === droppableId);
```

Comment:
javascript has a native `.filter` function -- it's preferred to use language features if they exist before using a separate library